src/app/components/Snapshots/SnapshotCardExternalLink.tsx

```diff
@@ -40,7 +41,7 @@ export const SnapshotCardExternalLink: FC<SnapshotCardExternalLinkProps> = ({
         >
           {description}
         </Typography>
-        {url && hasValidProtocol(url) && (
+        {url && (hasValidProtocol(url) || url.startsWith('mailto:')) && (
```

Comment:
> (unimportant note: you are not using emailAccepted)

This part has been fixed.

Reply:
> I don't think this is a good solution

What would you consider a good solution, instead of prop for the general purpose component?

Maybe a separate component used for displaying safe links?